KinectIPD/KinectIPD/App.xaml.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace KinectIPD
```

Comment:
NIT: I wonder if we should standardize on HoloLensCompanionKit namespaces.
Something like: 
namespace HoloLensCompanionKit.KinectIPD
namespace HoloLensCompanionKit.Samples
namespace HoloLensCompanionKit.Utilities etc.